src/app/project/ontology/default-data/default-properties.ts

```diff
@@ -145,13 +162,21 @@ export class DefaultProperties {
             elements: [
                 {
                     icon: 'link',
-                    label: 'Other resource e.g. Person',
+                    label: 'Resource class',
                     subPropOf: Constants.HasLinkTo,
                     objectType: Constants.LinkValue,
                     guiEle: Constants.SalsahGui + Constants.HashDelimiter + 'Searchbox',    // 'Autocomplete',
                     group: 'Link'
                 },
                 // {
+                //     icon: 'picture_in_picture',
```

Comment:
maybe yes. I had an ontology with a property "is part of", which is kind of link to another resource. This is something we have to discuss. And in the documentation I saw "link to external resource". But at the moment this property type is not yet fully implemented (because it's not an important one).